.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: release-rz-bindgen
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+
+jobs:
+  run-bindgen:
+    name: Run bindgen
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-latest]
+
+    steps:
+    - name: Checkout rz-bindgen
+      uses: actions/checkout@v3
+
+    - name: Checkout rizin
+      uses: actions/checkout@v3
+      with:
+        repository: rizinorg/rizin
+        path: rizin
+        ref: ${{ github.ref }}
+
+    - name: Run rz-bindgen
+      uses: pypa/cibuildwheel@v2.8.0
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        path: wheelhouse/*.whl
+
+  publish-pypi:
+    name: Publish PyPi
+    runs-on: ubuntu-latest
+    needs: [run-bindgen]
+    steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
+      with:
+        name: artifact
+        path: dist
+    - name: Publish wheels
+      uses: pypa/gh-action-pypi-publish@v1.5.1
```

Comment:
Same

Reply:
I'm not sure about these. I see also the doc says to use the full version. I'd keep them as they are for now.